resources/public/openapi/spec.yml

```diff
@@ -85,17 +85,73 @@ paths:
         401:
           description: User could not be authenticated.
 
-  /divisions:
+  /competitions:
     get:
-      tags: [divisions]
+      tags: [competitions]
+      summary: Get all competitions
+      responses:
+        200:
+          description: Successfully fetched competitions.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Competitions'
+    
+    post:
+      tags: [competitions]
+      summary: Create new competition
+      security:
+        - bearer: []
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Competition'
+      responses:
+        201:
+          description: Successfully created competitions.
+        400:
+          description: Failed to create competition.
+
+  /competitions/{competition}:
+    get:
+      tags: [competitions]
+      summary: Get competition by ID
+      parameters:
+        - in: path
+          name: competition
+          required: true
+          schema:
+            type: integer
+          description: ID for competition to fetch.
+      responses:
+        200:
+          description: Successfully retrieved competition.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Competition'
+        404:
+          description: Competition does not exist.
+
+  /competitions/{competition}/divisions:
+    get:
+      tags: [competitions]
       summary: Get all divisions or fetch division by key
       description: >
-        Can be used to either fetch all divisions or to find divisions by key.
+        Can be used to either fetch all divisions in a competition or to find
+        divisions by key.
 
         Note that the key is unique for divisions and therefore a singular
```

Comment:
Note: I'll probably change this to always return a list, even when guaranteed a singular value.